src/app/App.js

```diff
@@ -57,93 +65,48 @@ const App = () => {
 
   return (
     <Router>
-      {dark ? (
-        <div className="main dark">
-          <Switch>
-            <Route path="/" exact>
-              <div className="App">
-                <ToastContainer limit={2} />
-                <Header rotate={rotate} changeMode={changeMode} />
-                <div className="colorCont flex">
-                  {data.map((gradient) => (
-                    <Gradient
-                      gradient={gradient}
-                      id={gradient.id}
-                      align={align}
-                      key={gradient.id}
-                    />
-                  ))}
-                </div>
-                <Footer />
-              </div>
-            </Route>
-            <Route path="/gradient_generator" exact>
-              <ToastContainer limit={2} />
-              <Header rotate={rotate} changeMode={changeMode} />
-              <GradientGenerator />
-              <Footer />
-            </Route>
-            <Route path="/gradient/:id" exact>
-              <ToastContainer limit={2} />
-              <Header rotate={rotate} changeMode={changeMode} />
-              <Fullpage data={data} align={align} />
-            </Route>
-            <Route path="/contributors" exact>
-              <Header rotate={rotate} changeMode={changeMode} />
-              <div className="contributors flex">
-                {data.map((contributor) => (
-                  <Contributors contributor={contributor} />
-                ))}
-              </div>
-              <Footer />
-            </Route>
-            <Route component={Notfound} />
-          </Switch>
-        </div>
-      ) : (
-        <div className="main light">
-          <Switch>
-            <Route path="/" exact>
-              <div className="App">
-                <ToastContainer limit={2} />
-                <Header rotate={rotate} changeMode={changeMode} />
-                <div className="colorCont flex">
-                  {data.map((gradient) => (
-                    <Gradient
-                      gradient={gradient}
-                      id={gradient.id}
-                      align={align}
-                      key={gradient.id}
-                    />
-                  ))}
-                </div>
-                <Footer />
-              </div>
-            </Route>
-            <Route path="/gradient_generator" exact>
+      <div className={clsx(dark ? 'main dark' : 'main light')}>
```

Comment:
I saw that only the classnames were conditional so I added a new tiny utility package to help with conditional classnames. More info about the package: https://github.com/lukeed/clsx#readme

Reply:
This is super cool! Thanks for the new knowledge!